python/hackathon_app/sheets.py

```diff
@@ -126,6 +128,21 @@ def rows(self) -> Sequence[TModel]:
             raise SheetError(str(ex))
         return response
 
+    def update(self, model: TModel, row: int):
```

Comment:
```suggestion
    def _update(self, model: TModel, row: int):
```

I think it's fine for now to have a base ("protected") update method and call it in child classes.